tools/code_format/check_format.py

```diff
@@ -806,7 +806,9 @@ def has_non_comment_throw(line):
             return False
 
         if self.deny_listed_for_exceptions(file_path):
-            if has_non_comment_throw(line) or "THROW" in line or "throwExceptionOrPanic" in line:
+            if has_non_comment_throw(
+                    line) or "THROW" in line or "throwEnvoyExceptionOrPanic" in line:
```

Comment:
nit: mho is that multi-line conditions are a readability anti-pattern - at least in python where space is significant